### DISCUSSION TITLE
Help needed with IF-ELSE

### DISCUSSION THREAD

**Discussion by skalozubio** [pawelsalawa/sqlitestudio#4359] [Q&A]
Hello. 
I'm making a mod for Civilization VI game. The mod translates english texts into Russian.
There is a Table named "LocalizedText". It has columns Tag and Text.
I need:
If Tag = 'LOC_DISTRICT_BATH_DESCRIPTION' AND Text LIKE '%district unique to Rome%'
Then Where Tag = 'LOC_DISTRICT_BATH_NAME' SET Text = 'Aqueduct'
I've tried plenty of times, but I'm new at SQL.
Could you plese help me.

**Comment by pawelsalawa** [comment]
You are making a condition on a `Tag` column twice. First for value `LOC_DISTRICT_BATH_DESCRIPTION` and second for `LOC_DISTRICT_BATH_NAME`. Which one should it be then?

If it was for the first value of tag, then query would look like:
```
UPDATE LocaLIzedText
   SET Text = 'Aqueduct'
 WHERE Tag = 'LOC_DISTRICT_BATH_DESCRIPTION'
   AND Text LIKE '%district unique to Rome%';
```

**Reply by skalozubio** [reply]
Yes, I need 2 conditions. At first, I should check is LOC_DISTRICT_BATH_DESCRIPTION contains '%district unique to Rome%'?
If yes, I should replace Anotjer tag's text -  'LOC_DISTRICT_BATH_NAME'
Is it possible?

**Reply by pawelsalawa** [reply]
This doesn't make sense.
Is the `Tag` a column, or are there multiple , different tag columns?
Can you show how this table looks like?

**Reply by skalozubio** [reply]
Tag and Text are columns. But, I should change one tag's text, if another tag's text contains needed value.
![Screenshot 2022-01-04 165729](https://user-images.githubusercontent.com/81754308/148049085-2bdb4cd5-6870-4a64-95ed-aecf162c52e8.png)

**Reply by pawelsalawa** [reply]
I see. In that case:

```
UPDATE LocalizedText
   SET Text = 'Aqueduct'
 WHERE Tag = 'LOC_DISTRICT_BATH_NAME'
   AND EXISTS (
            SELECT 1
              FROM LocalizedText
             WHERE Text LIKE '%district unique to Rome%'
               AND Tag = 'LOC_DISTRICT_BATH_DESCRIPTION'
       );
```

**Reply by skalozubio** [reply]
Oh! Thank you very much!!! It's exactly what I needed!
